x-pack/plugins/cases/server/connectors/cases/cases_connector_executor.ts

```diff
@@ -752,28 +746,44 @@ export class CasesConnectorExecutor {
     };
   }
 
-  private getCasesTitle(ruleName: string, oracleCounter: number) {
-    const suffix =
-      oracleCounter === INITIAL_ORACLE_RECORD_COUNTER
-        ? '(Auto-created)'
-        : `(${oracleCounter}) (Auto-created)`;
+  private getCasesTitle(
+    params: CasesConnectorRunParams,
+    grouping: GroupedAlerts['grouping'],
+    oracleCounter: number
+  ) {
+    const groupingDescription = Object.entries(grouping)
+      .map(([_, value]) => {
+        return convertValueToString(value);
+      })
+      .join(' & ');
 
-    const ruleNameTrimmed = ruleName.slice(0, MAX_TITLE_LENGTH - suffix.length - 1);
+    const suffix = `${
+      groupingDescription.length > 0 ? ` - Grouping by ${groupingDescription}` : ''
+    }${oracleCounter > INITIAL_ORACLE_RECORD_COUNTER ? ` (${oracleCounter})` : ''} (Auto-created)`;
 
-    return `${ruleNameTrimmed} ${suffix}`;
+    const ruleNameTrimmed = params.rule.name.slice(0, MAX_TITLE_LENGTH - suffix.length - 1);
+
+    return `${ruleNameTrimmed}${suffix}`;
   }
 
-  private getGroupingDescription(grouping: GroupedAlerts['grouping']) {
-    const flattenGrouping = getFlattenedObject(grouping);
+  private getCaseDescription(params: CasesConnectorRunParams, grouping: GroupedAlerts['grouping']) {
+    const ruleName = params.rule.ruleUrl
+      ? `[${params.rule.name}](${params.rule.ruleUrl})`
+      : params.rule.name;
 
-    return Object.entries(flattenGrouping)
-      .map(([key, value]) => {
-        const keyAsCodeBlock = `\`${key}\``;
-        const valueAsCodeBlock = `\`${convertValueToString(value)}\``;
+    const description = `This case was created by the Case action in ${ruleName}.`;
```

Comment:
```suggestion
    const description = `This case was created by the case action in ${ruleName}.`;
```

Reply:
Per https://eui.elastic.co/#/display/toast/guidelines there is a common pattern for using the object type then object name (in single quotes). So something like "This case was created by the case action in rule 'Rule name'." would follow that pattern. Or maybe you don't even need to mention the case action. For example: "This case was created by the rule 'Rule name'."

Reply:
Done in [`18cf2d2` (#180758)](https://github.com/elastic/kibana/pull/180758/commits/18cf2d2d810624e6815cca0bf5826f849d43b005)